lessons/cryptography/Encoding/encoding.md

```diff
@@ -0,0 +1,87 @@
+# [Cryptography] Encoding
+*Learn about encoding, a way to reformat information.*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
```

Comment:
Let's please make the objectives more about the process of encoding than specific encodings. Although I think common specific encodings are still a good thing to talk about.

Reply:
I think this is still relevant. We need to have broader objectives as this is a 100-level lesson.

Reply:
updated